### PR TITLE
Replace nevow with twisted.web.template in status.PublishStatusPage

### PR DESCRIPTION
**Pull request by sajith**
Fixes [3289](https://tahoe-lafs.org/trac/tahoe-lafs/ticket/3289).  

For comparison, screenshot of the page after this change is below below:

wui-publish-status-after

-----

Before:

wui-publish-status-before

### CODE REVIEW

File: src/allmydata/web/status.py [hunk 1]
```diff
@@ -759,114 +759,162 @@ def render_server_timings(self, ctx, data):
         return T.li["Per-Server Fetch Response Times: ", l]
 
 
-class PublishStatusPage(rend.Page, RateAndTimeMixin):
-    docFactory = getxmlfile("publish-status.xhtml")
+# Renders "status/publish-%n"
```
Comment (meejah) [inline src/allmydata/web/status.py:762]: as a docstring instead would be better

File: src/allmydata/web/status.py [hunk 2]
```diff
@@ -759,114 +759,162 @@ def render_server_timings(self, ctx, data):
...
+class PublishStatusPage(MultiFormatResource):
 
-    def __init__(self, data):
-        rend.Page.__init__(self, data)
-        self.publish_status = data
+    def __init__(self, publish_status):
+        """
+        :publish_status mutable.publish.PublishStatus: publish stats provider.
```
Comment (meejah) [inline src/allmydata/web/status.py:767]: `:param mutable.publish.PublishStatus publish_status: ...`

File: src/allmydata/web/status.py (hunk 1 above)
Comment (sajith) [reply src/allmydata/web/status.py]: Thank you!  Changed in 2e0e210.

File: src/allmydata/web/status.py (hunk 2 above)
Comment (sajith) [reply src/allmydata/web/status.py]: Updated in ebe8022.

If only pydocstyle wasn't so noisy... :-)

**Pull request merged**